test/data/migration-config.yml

```diff
@@ -1,3 +1 @@
-migration:
-  target:
-    product: foo
+migration_product: 'foo'
```

Comment:
If you like you can use a really existing product name in the test here

```yaml
migration_product: 'SLES/15/x86_64'
```

Reply:
I agree we should use an actual value such that it functions as an example for the value we expect.